app/models/mhv_account.rb

```diff
@@ -56,8 +67,11 @@ class MhvAccount < ActiveRecord::Base
   end
 
   def create_and_upgrade!
-    create_mhv_account! unless preexisting_account?
-    upgrade_mhv_account!
+    # Accept T&C if existing?
```

Comment:
Should we persist any T&C acceptance info if the account is preexisting? Probably not?

Reply:
I think we should be OK on that. Technically the T&C will hopefully get moved to the account creation controller anyway and would no longer need to be tied to an mhv_account state at all in the future.